### DISCUSSION TITLE
Welcome to 1build Discussions!

### DISCUSSION THREAD

**Discussion by gopinath-langote** [gopinath-langote/1build#200] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by i6basaffar** [comment]
Hi

**Reply by gopinath-langote** [reply]
hi

**Reply by i6basaffar** [reply]
@

**Reply by i6basaffar** [reply]
O references
private void OnCounterClicked(obiect sender, EventArgs e)
{
E% NewButton.Parent| null
count++:
var test = NewButton;
E& NewBorder.Parent QView - (Microsoft.Maui.Controls. Grid}
var test2 = NewBorder;
SemanticScreenReader.Announce(CounterLabel.Text);
string msg
= string.Empty;
if (sender is Button button)
